docs/source/references/api_spec.md

```diff
@@ -251,26 +256,7 @@ Also returns current status of peer in json string:
 **Expects**: -
 
 **Responses**:
-- 200 OK - currently mirrors status:
```

Comment:
I would not remove this. There's a reeason why Prometheus returns values in a human (rather than machine) readable format.